### PR TITLE
fix: App group deployment status

### PR DESCRIPTION
**Pull request by milindk8**
# Description

- Icon & status showing healthy instead of succeeded
- Icon size fix
- Breadcrumb naming

Fixes [#AB2143](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/2143)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
by observing statuses & icons in the app grouping overview

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/Environments/Constants.ts [hunk 1]
```diff
@@ -95,13 +95,18 @@ export const BULK_CD_MESSAGING = {
...
     SUBTITLE: 'You don’t have access to any application in this app group.',
     NO_MATCHING_ENV: 'No matching env',
     NO_MATCHING_RESULT: 'We couldn’t find any matching results',
     EMPTY_ENV: 'Empty environment',
 }
 
-export const NO_ACCESS_TOAST_MESSAGE = 'You don’t have access to any application in this app group'
+export const NO_ACCESS_TOAST_MESSAGE = {
+    
+    forSuperAdmin: 'There are no applications in this application group.',
```
Comment (shivani170) [inline src/components/Environments/Constants.ts:107]: Naming must be fixed

File: src/components/Environments/EnvironmentDetailsRoute.tsx [hunk 2]
```diff
@@ -12,11 +12,11 @@ import EnvironmentOverview from './EnvironmentOverview/EnvironmentOverview'
 import { EnvSelector } from './EnvSelector'
 import ResourceListEmptyState from '../ResourceBrowser/ResourceList/ResourceListEmptyState'
 import EmptyFolder from '../../assets/img/Empty-folder.png'
-import { EMPTY_LIST_MESSAGING, ENV_APP_GROUP_GA_EVENTS } from './Constants'
+import { EMPTY_LIST_MESSAGING, ENV_APP_GROUP_GA_EVENTS, NO_ACCESS_TOAST_MESSAGE } from './Constants'
 import { EnvHeaderType } from './EnvironmentGroup.types'
 import { ReactComponent as Settings } from '../../assets/icons/ic-settings.svg'
 
-export default function EnvironmentDetailsRoute() {
+export default function EnvironmentDetailsRoute({isSuperAdmin}) {
```
Comment (shivani170) [inline src/components/Environments/EnvironmentDetailsRoute.tsx:19]: Type missing

File: src/components/common/ToastBody.tsx [hunk 3]
```diff
@@ -51,10 +51,10 @@ export class ToastBodyWithButton extends React.Component<{
     }
 }
 
-export const toastAccessDenied = (subtitle?: string) => {
+export const toastAccessDenied = (title?: string, subtitle?: string) => {
     return toast.info(
         <ToastBody
-            title="Access denied"
+            title={title || "Access denied"}
```
Comment (shivani170) [inline src/components/common/ToastBody.tsx:57]: Move these messages to constant

File: src/components/Environments/EnvironmentsRoute.tsx [hunk 4]
```diff
@@ -5,18 +5,18 @@ import EnvironmentsList from './List/EnvironmentsList'
 import EnvironmentDetailsRoute from './EnvironmentDetailsRoute'
 import { AppContext } from '../common'
 
-export default function EnvironmentsRoute() {
+export default function EnvironmentsRoute({isSuperAdmin}) {
```
Comment (shivani170) [inline src/components/Environments/EnvironmentsRoute.tsx:8]: Type is missing

File: src/components/ApplicationGroup/List/EnvironmentsList.tsx [hunk 5]
```diff
@@ -11,7 +11,7 @@ import { Cluster } from '../../../services/service.types'
 import { AppListConstants } from '../../../config'
 import { useHistory, useLocation } from 'react-router-dom'
 
-export default function EnvironmentsList() {
+export default function EnvironmentsList({isSuperAdmin}:{isSuperAdmin: boolean}) {
```
Comment (vivek-devtron) [inline src/components/ApplicationGroup/List/EnvironmentsList.tsx:14]: Please move this to type also format the file.

File: src/components/ApplicationGroup/AppGroupRoute.tsx [hunk 6]
```diff
@@ -2,21 +2,21 @@ import React, { useState } from 'react'
 import { URLS } from '../../config'
 import { Redirect, Route, Switch, useRouteMatch } from 'react-router-dom'
 import EnvironmentsList from './List/EnvironmentsList'
-import EnvironmentDetailsRoute from './EnvironmentDetailsRoute'
+import EnvironmentDetailsRoute from './AppGroupDetailsRoute'
 import { AppContext } from '../common'
 
-export default function EnvironmentsRoute() {
+export default function EnvironmentsRoute({isSuperAdmin}:{isSuperAdmin: boolean}) {
```
Comment (vivek-devtron) [inline src/components/ApplicationGroup/AppGroupRoute.tsx:8]: Please move this to type and format the file

File: src/components/ApplicationGroup/AppGroupDetailsRoute.tsx [hunk 7]
```diff
@@ -6,17 +6,17 @@ import ReactGA from 'react-ga4'
...
 import EmptyFolder from '../../assets/img/Empty-folder.png'
-import { EMPTY_LIST_MESSAGING, ENV_APP_GROUP_GA_EVENTS } from './Constants'
-import { EnvHeaderType } from './EnvironmentGroup.types'
+import { EMPTY_LIST_MESSAGING, ENV_APP_GROUP_GA_EVENTS, NO_ACCESS_TOAST_MESSAGE } from './Constants'
 import { ReactComponent as Settings } from '../../assets/icons/ic-settings.svg'
+import { getEnvAppList } from './AppGroup.service'
+import { EnvHeaderType } from './AppGroup.types'
 
-export default function EnvironmentDetailsRoute() {
+export default function EnvironmentDetailsRoute({isSuperAdmin}:{isSuperAdmin: boolean}) {
```
Comment (vivek-devtron) [inline src/components/ApplicationGroup/AppGroupDetailsRoute.tsx:19]: Please move this to type and format the file

File: src/components/ApplicationGroup/List/EnvironmentsList.tsx (hunk 5 above)
Comment (milindk8) [reply src/components/ApplicationGroup/List/EnvironmentsList.tsx]: done

File: src/components/ApplicationGroup/AppGroupRoute.tsx (hunk 6 above)
Comment (milindk8) [reply src/components/ApplicationGroup/AppGroupRoute.tsx]: done

File: src/components/ApplicationGroup/AppGroupDetailsRoute.tsx (hunk 7 above)
Comment (milindk8) [reply src/components/ApplicationGroup/AppGroupDetailsRoute.tsx]: done

File: src/components/ApplicationGroup/List/EnvironmentsList.tsx [hunk 8]
```diff
@@ -10,8 +10,9 @@ import { getClusterListMinWithoutAuth } from '../../../services/service'
 import { Cluster } from '../../../services/service.types'
 import { AppListConstants } from '../../../config'
 import { useHistory, useLocation } from 'react-router-dom'
+import { AppGroupAdminType } from '../AppGroup.types'
 
-export default function EnvironmentsList() {
+export default function EnvironmentsList({ isSuperAdmin }: AppGroupAdminType) {
```
Comment (shivani170) [inline src/components/ApplicationGroup/List/EnvironmentsList.tsx:15]: Why the type name isn't resembling component name?

**Review by vivek-devtron — APPROVED** [review]

**Review by shivani170 — APPROVED** [review]

**Pull request merged**